articles/150_ros_command_line.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: default
+title: ROS Command Line
+permalink: articles/ros_command_line.html
```

Comment:
Change to: `ros_command_line_arguments`

Reply:
Changed in 4c1b29833c98640308a183a28d9a9b35e8ed1ca5.